src/components/KExternalLink/KExternalLink.vue

```diff
@@ -45,7 +45,6 @@ const isHrefValid = computed((): boolean => !!isValidUrl(props.href))
   font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
   gap: var(--kui-space-20, $kui-space-20);
   list-style: inherit;
-  outline: none;
```

Comment:
Removing because as outlined in one of the accessibility findings from the recent Konnect audit by Level Access, link focus indicator must be visible -- while we don't always control where this component is used so it's best that we combine native browser outline with custom state styling.